e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/keyboardNavigation.functional.ts

```diff
@@ -4729,3 +4729,56 @@ test('DataGrid - Data rows are skipped during Tab navigation if the first column
   showBorders: true,
   width: 300,
 }));
+
+test('DataGrid input cell should not put tabindex to incorrect element while on edit mode (T1239462)', async (t) => {
+  const dataGrid = new DataGrid('#container');
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const button = Selector('.dx-datagrid-addrow-button');
```

Comment:
We should avoid hardcoded selectors. You can operate with toolbar in [the following manner](https://github.com/DevExpress/DevExtreme/blob/d99def29a67457fdc44e202b0d2a4dcbd3dbb13d/e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/keyboardNavigation.functional.ts#L4302).

Reply:
Thank you!